docs/Marketplace.md

```diff
@@ -338,62 +412,62 @@ curl 'http://localhost:8000/api/codex/v1/sales/availability' \
 }'
 ```
 
-This should return a response with an id a string (e.g. `"id": "0x552ef12a2ee64ca22b237335c7e1df884df36d22bfd6506b356936bc718565d4"`) which identifies this storage offer. To check the current storage offers for this node, you can issue:
+This should return a JSON response containing an `id` (e.g. `"id": "0x552ef12a2ee64ca22b237335c7e1df884df36d22bfd6506b356936bc718565d4"`) which identifies this storage offer. To check the current storage offers for this node, you can issue:
 
 ```bash
-curl 'http://localhost:8000/api/codex/v1/sales/availability'
+curl 'http://localhost:8000/api/codex/v1/sales/availability'  | jq
 ```
 
-This should print a list of offers, with the one you just created figuring among them.
+> `jq` is a handy formatting utility that will make reading the JSON response easier. On macOS you can install with `brew install jq`.
+
+This should print a list of offers, with the one you just created figuring among them (for our tutorial, there will be only one offer returned at this time).
 
 ## 4.2. Buy Storage
 
-Before we can buy storage, we must have some actual data to request storage for. Start by uploading a small file to your client node. On Linux you could, for instance, use `dd` to generate a $100KB$ file:
+Before we can buy storage, we must have some actual data to request storage for. Start by uploading a small file to your client node. On Linux (or macOS) you could, for instance, use `dd` to generate a $1M$ file:
 
 ```bash
-dd if=/dev/urandom of=./data.bin bs=100K count=1
+dd if=/dev/urandom of=./data.bin bs=1M count=1
 ```
 
-but any small file will do. Assuming your file is named `data.bin`, you can upload it with:
+~but any small file will do.~
+
+> @MC: I am not sure if any small file will do, still do not understand the exact logic behind, but when using a small file (say `100KB`), I was getting the following error: _Cannot setup slots for a dataset with ecK == numBlocks. Please use a larger file or a different combination of `nodes` and `tolerance`._ When I changed the file size to something slightly bigger (like `1MB`), I was way more consistently successful.
+
```

Comment:
Yeah this was because of a bug we had, which got corrected. I'm gonna add a follow-up issue to test and lift that restriction.

Reply:
Follow-up issue: https://github.com/codex-storage/nim-codex/issues/889